apps/base-docs/docs/connecting-to-the-blockchain/overview-of-providers.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Overview of Providers
+description: Learn what providers are and why you need one.
+hide_table_of_contents: false
+---
+
+Onchain apps need frontends, sometimes called dApps, to enable your users to interact with your smart contracts. A _provider_ makes the connection from frontend to blockchain, and is used to read data and send transactions.
+
+---
+
+## Prerequisites
+
+Before this lesson, you should:
+
+- Be familiar with modern, frontend web development
+- Possess a general understanding of the EVM and smart contracts
+- Ideally, understand how to build a frontend with [Next.js]
+
+---
+
+## Objectives
+
+By the end of this lesson you should be able to:
+
+- Compare and contrast public providers vs. vendor providers vs. wallet providers
+- Select the appropriate provider for several use cases
+
+---
+
+## Types of Providers
+
+In blockchain development, the term _provider_ describes a company or service that provides an API enabling access to the blockchain as a service. This is distinct from the provider in the the React Context API or Redux Provider, though you will also wrap your app with `wagmiConfig` in a similar manner.
+
+These services enable interacting with smart contracts without the developer needing to run and maintain their own blockchain node. Running a node is expensive, complicated, and challenging. In most cases, you'll want to start out with a provider. Once you start to get traction, you can evaluate the need to run your own node, or switch to a more advanced architecture solution, such as utilizing [Subgraph].
+
+Figuring out which type of provider to use can be a little confusing at first. As with everything blockchain, the landscape changes rapidly and search results often return out-of-date information.
+
+:::info
+
+New onchain devs sometimes get the impression that there are free options for connecting their apps to the blockchain. Unfortunately, this is not really true. Blockchain data is still 1's and 0's, fetched by computation and served to the internet via servers.
+
+It costs money to run these and you will eventually need to pay for the service.
+
+:::
+
+You'll encounter providers divided into three general categories: Public Providers, Wallet Providers, and Vendor Providers
+
+### Public Providers
+
+Many tutorials and guides, as well getting started guides for libraries such as [wagmi], will use a _Public Provider_ as the default to get you up and running. Public means that they're open, permissionless, and free, so the guides will also usually warn you that you need to add another provider if you don't want to run into rate limiting. Listen to these warnings! The rate-limits of public providers are severe and you'll start getting limited very quickly.
```

Comment:
Consider - `libraries such as wagmi` -> `third-party client libraries, such as [ethers.js](link to our client ether.js documentation)`

I feel like ether.js is a more basic / foundational example of a library for reading blockchain data, whereas wagmi is specific to React devs looking for hooks and handling wallet connection. (and we have documentation we can point to for ethers.js)